Foundation/NSDecimal.swift

```diff
@@ -1479,26 +1566,6 @@ fileprivate struct WideDecimal : VariableLengthNumber {
             }
         }
     }
-    public func compare(to rhs: WideDecimal) -> ComparisonResult {
```

Comment:
This was added as part of a transitional implementation in the previous commit, but is no longer necessary - so it has been removed.